connect/api/src/main/java/org/apache/kafka/connect/data/ConnectSchema.java

```diff
@@ -93,6 +93,7 @@
     // Optional human readable documentation describing this schema.
     private final String doc;
     private final Map<String, String> parameters;
+    private final int hc;  // precomputed hash code
```

Comment:
Traditionally we don't overuse acronyms for variables. Should we call it `hash` or `hashCode` to make really obvious what this variable stands for? Also, to be even more _nitpicky_ given the small range of changes, I believe we add a comment on its own line (see above comment line for `doc` variable).